SlevomatCodingStandard/Sniffs/Namespaces/FullyQualifiedClassNameInAnnotationSniff.php

```diff
@@ -23,6 +25,9 @@ class FullyQualifiedClassNameInAnnotationSniff implements Sniff
 
 	public const CODE_NON_FULLY_QUALIFIED_CLASS_NAME = 'NonFullyQualifiedClassName';
 
+	/** @var list<string> */
+	public $ignoredAnnotationNames = [];
```

Comment:
Please add the option to sniff documentation.

Reply:
you mean in `namespaces.md` file?